.gitignore

```diff
@@ -26,3 +26,4 @@ db/data.yml
 nbproject
 rerun.txt
 tags
+*.iml
```

Comment:
Can you add this to your system ignore rules instead of the project's `.gitignore`? Git is prescriptive on this point: https://git-scm.com/docs/gitignorehttps://git-scm.com/docs/gitignore